CONTRIBUTING.md

```diff
@@ -37,7 +37,22 @@ license.
 
 ### Submitting a contribution, opening an issue.
 
-Please use the github standard interfaces to do so.
+Please use the github standard interfaces to do so. When you submit a
+pull-request, you agree to share your code under `coq-lsp` license.
+
+We have a set of tags to classify pull requests and issues, we try to
+use them as much as possible.
+
+### Changelog
+
+We require an entry in `CHANGES.md` for all user-impacting changes;
+note that as of today, `CHANGES.md` is the canonical changelog
+including _both_ server and client items.
+
+The client changelog that is used by the VS Code marketplace is at
+`editor/code/CHANGELOG.md`, but you should not modify it, as of today
```

Comment:
We should put a html comment in that changelog at the top telling poeple not to modify.

Reply:
Good idea, thanks.

Reply:
However it is a bit tricky update that in release mode, mmmm

Reply:
Skipped that for now, as it is too ackward, hopefully can be improved.